pkg/test/utils/kubernetes.go

```diff
@@ -224,7 +224,7 @@ func (r *RetryClient) Patch(ctx context.Context, obj client.Object, patch client
 // Get retrieves an obj for the given object key from the Kubernetes Cluster.
 // obj must be a struct pointer so that obj can be updated with the response
 // returned by the Server.
-func (r *RetryClient) Get(ctx context.Context, key client.ObjectKey, obj client.Object) error {
+func (r *RetryClient) Get(ctx context.Context, key client.ObjectKey, obj client.Object, opts ...client.GetOption) error {
 	return Retry(ctx, func(ctx context.Context) error {
 		return r.Client.Get(ctx, key, obj)
```

Comment:
Shouldn't `opts` be passed to the `r.Client.Get` call like the CRUD functions?

Reply:
double checking... thanks for the watchful eye